src/commands/sarif/README.md

```diff
@@ -32,9 +32,9 @@ The positional arguments are the directories or file paths in which the SARIF re
 
 Additionally, you may configure the `sarif` command with environment variables:
 
-- `DATADOG_API_KEY` or `DD_API_KEY` (**required**): API key used to authenticate the requests.
+- `DD_API_KEY` (**required**): API key used to authenticate the requests.
```

Comment:
Agreed, the command already supported both, and let's only document `DD_API_KEY` and `DD_SITE` 👍